src/app/shared/logger.service.ts

```diff
@@ -1,4 +1,4 @@
-import { Injectable } from '@angular/core';
+import { Injectable } from "@angular/core";
```

Comment:
This is another change that is not related to the PR. Additionally, in most of the files we are using single quotes. Why would you change this?